v1/ao/internal/reporter/reporter_grpc.go

```diff
@@ -829,7 +829,7 @@ func (r *grpcReporter) collectMetrics(collectReady chan bool) {
 	i := atomic.LoadInt32(&r.collectMetricInterval)
 	// generate a new metrics message
 	builtin := metrics.BuildBuiltinMetricsMessage(r.httpMetrics.CopyAndReset(i),
-		r.eventConnection.queueStats.CopyAndReset(), FlushRateCounts())
+		r.eventConnection.queueStats.CopyAndReset(), FlushRateCounts(), config.GetRuntimeMetrics())
```

Comment:
I assume this will be `false` by default? Do we want to make it `true`?

Reply:
Yes it's false by default. From what I could remember, it's because the runtime metrics are counted as custom metrics and some customers may not needed by default.

Reply:
It is true...hmm...for JMX it did get enabled by default even though that means using up some of the metrics quota. @cheempz what do u think? :)

Reply:
We discussed this in the proposal/spec for go runtime metrics and decided it's safer to disable them by default; this preserves current behaviour for customers (go agent is sending them but they are dropped by collector).  Can revisit once there's an automatic dashboard created similar to JMX. cc @adam_hert

Reply:
Hey Adam commented in the Jira that it's OK to have this enabled by default!

Reply:
Fixed: flipped the default value to `true`